Add LTLIBICONV to popt.pc.in

Add ${LTLIBICONV} to popt.pc.in so applications such as shairport-sync
will know that they must link with -liconv when building statically

Fixes:
 - http://autobuild.buildroot.org/results/c5b0d1d2867e49c022a2ad971dd9f358ff0f3865

Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>

diff -Naurp popt-1.18-vanilla/popt.pc.in popt-1.18/popt.pc.in
--- popt-1.18-vanilla/popt.pc.in	2020-03-24 10:36:25.000000000 +0100
+++ popt-1.18/popt.pc.in	2020-09-18 21:18:03.112609343 +0200
@@ -7,4 +7,5 @@ Name: popt
 Version: @VERSION@
 Description: popt library.
 Libs: -L${libdir} -lpopt
+Libs.private: @LTLIBICONV@
 Cflags: -I${includedir}
